catroid/src/androidTest/java/org/catrobat/catroid/test/content/sprite/SpriteTest.java

```diff
@@ -114,4 +120,16 @@ public void testUserVariableVisibilityOfLocalVariablesInDifferentScenes() {
 		userVariable = sprite2.getUserVariable(variableName);
 		assertTrue(userVariable.getVisible());
 	}
+
+	@Test
+	public void spriteEqualBackgroundTest() {
+		sprite.setName("Hintergrund");
+		updateLocale(ApplicationProvider.getApplicationContext(), new Locale("de"));
+		setLanguageSharedPreference(ApplicationProvider.getApplicationContext(), "de");
+		project.checkIfSpriteNameEqualBackground(ApplicationProvider.getApplicationContext());
+		Assert.assertNotEquals(sprite.getName(), "Hintergrund");
+		Assert.assertEquals(sprite.getName(), "Hintergrund (1)");
+		updateLocale(ApplicationProvider.getApplicationContext(), new Locale("en-gb"));
```

Comment:
Is en-GB really the default locale and not en-US?

Also, shouldn't we just reset the language to the previously set language and not hard-code it here?